lib/web_components_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module WebComponentsRails
-  VERSION = '2.1.2'
+  VERSION = '2.1.3'
```

Comment:
I think this deserves a bump to 2.2.0. Not that this is that widely used, but this is a decent sized changed.